pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/exception/DuplicateException.java

```diff
@@ -5,4 +5,8 @@ public class DuplicateException extends RuntimeException {
     public DuplicateException(String message) {
         super(message);
     }
+
+    public DuplicateException(String message, Exception cause) {
```

Comment:
Pretty much any exception class we create should have the `(message,cause)` constructor in addition to just the `(message)` constructor, since our exceptions tend to wrap other exceptions.

Reply:
See my comment below, but I'd just as soon get rid of DuplicateException rather than extend it.